Lib/fontbakery/profiles/stat.py

```diff
@@ -0,0 +1,31 @@
+from fontbakery.callable import check
+from fontbakery.checkrunner import FAIL, PASS, WARN
+from fontbakery.message import Message
+# used to inform get_module_profile whether and how to create a profile
+from fontbakery.fonts_profile import profile_factory # NOQA pylint: disable=unused-import
+
+profile_imports = [
+    ('.shared_conditions', ('is_variable_font',))
+]
+
+@check(
+    id = 'com.google.fonts/check/varfont/stat_axis_record_for_each_axis',
+    rationale = """
+      According to the OpenType spec, there must be an Axis Record for every axis defined in the fvar table, https://docs.microsoft.com/en-us/typography/opentype/spec/stat#axis-records
+    """,
+    conditions = ['is_variable_font']
+)
+def com_google_fonts_check_varfont_stat_axis_record_for_each_axis(ttFont):
+    """According to the OpenType spec, there must be an Axis Record for every
+    axis defined in the fvar table."""
+    fvar_axes = set(a.axisTag for a in ttFont['fvar'].axes)
+    stat_axes = set(a.AxisTag for a in ttFont['STAT'].table.DesignAxisRecord.Axis)
+    missing_axes = fvar_axes - stat_axes
+    if len(missing_axes) > 0:
+        yield FAIL, Message("missing-axis-records", (
+            "STAT table is missing Axis Records for the following "
+            "axes {missing_axes}")
+        )
```

Comment:
you forgot to use the f-string prefix here.

Other than that, this does not follow our code style. I know that this is completely arbitrary, but for the sake of code-style consistency, this is the style we have been adopting:

```python
yield FAIL,\
      Message("missing-axis-records",
              f"STAT table is missing Axis Records for the"
              f" following axes {missing_axes}")
```

Please observe the main aspects of this code-style and some reasoning behind it:
* Line break after comma / before `Message`
* Keep `Message` visually aligned to `FAIL` (or whatever else status code)
* When breaking strings, please leave the space char in the beginning of the next line, instead of at the end of the previous line
* Do not use parentheses around multi-line strings (unless strictly necessary to comply with python syntax. Not needed here and by not using them we avoid clutter)
* The lonely closing parentheses annoys me a bit, so keep the code more compact by placing it in the same line as the closing double-quotes
* the `f` prefix on f-strings would not be necessary in all lines of a multi-line string, but just those where variables are referenced. Regardless of that, we have been using the prefix in all lines just because it looks nicer.